src/components/Modal/_modal.scss

```diff
@@ -20,6 +20,9 @@
     &.--is-open {
       background-color: rgba(23, 23, 23, 0.5);
       z-index: 9999;
+      &.#{$prefix}--tooltip {
```

Comment:
Assuming this is so the tooltip doesn't render behind the modal?

Reply:
Sorry, I forgot to remove this one. But yeah, tooltips in modals are being rendered behind the overlay, that why I added the z-index in the index.scss.